app/routes/__index.tsx

```diff
@@ -0,0 +1,13 @@
+import { Outlet } from '@remix-run/react';
+import Header from '~/components/header';
+
+export default function Index() {
+  return (
+    <div>
```

Comment:
I recall we're using chakra component as much as possible, should we be using the `Box` component instead of div?

Reply:
I can change this to a box, sure